libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -150,6 +150,7 @@ class AP_SerialManager {
         SerialProtocol_AirSpeed = 34,
         SerialProtocol_ADSB = 35,
         SerialProtocol_AHRS = 36,
+        SerialProtocol_FETtechOneWire = 37,
```

Comment:
This also needs to be added to the parameter values list, right?

Reply:
yep.